onedal/__init__.py

```diff
@@ -34,7 +34,10 @@
             dal_root_redist = os.path.join(os.environ["DALROOT"], "redist", arch_dir)
             if os.path.exists(dal_root_redist):
                 os.add_dll_directory(dal_root_redist)
-        os.add_dll_directory(path_to_libs)
+        try:
+            os.add_dll_directory(path_to_libs)
+        except FileNotFoundError:
+            pass
```

Comment:
Does it make sense to add warning here?

Reply:
the dll directory add makes sure that the conda-installed onedal/tbb/mkl shared objects are properly in the windows path for python to discover and include.  It will fail if the folder doesn't exist, but that doesn't mean that the shared objects aren't discoverable.  If the various components like onedal and tbb are manually activated, then everything should run as normal (as is the case in the CI system).  We can test for the case if onedal is installed via pip, but that would require a different CI setup, and could possibly be an extension similar to #1992